manim/cli/init/commands.py

```diff
@@ -0,0 +1,28 @@
+"""Manim's init subcommand.
+
+Manim's init subcommand is accessed in the command-line interface via ``manim
+init``. Here you can specify options, subcommands, and subgroups for the init
+group.
+
+"""
+from pathlib import Path
+
+import click
+
+from ...constants import CONTEXT_SETTINGS, EPILOG
+from ...utils.file_ops import copy_template_files
+
+
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+    no_args_is_help=False,
+    epilog=EPILOG,
+    help="Sets up a project in current working directory with default settings.",
```

Comment:
Using this `help` option overwrites the content of the docstring down below. You should prefer the docstring down below so that you can be more descriptive in the help message.